src/main/java/codeu/model/data/NemoBot.java

```diff
@@ -22,7 +23,7 @@
   /**
   * Constructs a new Bot.
   */
-  public NemoBot(String filePath) {
```

Comment:
Why the changes in this file? We are actually passing the file path not the file right? I'd say just re-checkout this file from master to revert all changes.